packages/evo-marko/src/tags/evo-icon-button/index.marko

```diff
@@ -0,0 +1,68 @@
+static const VALID_SIZES = ["small", "large"];
+import type { AttrString } from "marko/tags-html";
+static var validPriorities = ["primary", "secondary", "tertiary"];
+import type { Input as ButtonInput } from "../evo-button/index.marko";
```

Comment:
It doesn't look like we're actually _using_ `evo-button` here so it doesn't feel right to be coupling the types like this... I think if it were up to me I'd just duplicate the `partiallyDisabled` and `onEscape` types

Reply:
I initially was going to extend Icon button from button, but there were some attributes we werent using. 
I think its fine as is. I did add priority and size as well